ToolbarManager/Patches/MyGuiScreenToolbarConfigBasePatch.cs

```diff
@@ -228,6 +228,7 @@ private static void OnStagingGridOnDrop(MyGuiControlGrid stagingGrid, MyDragAndD
             {
                 // Clone the item to remove the toolbar number and current state, which is not relevant in the staging area
                 var clone = new MyGuiGridItem(item.Icons, item.SubIcon, item.ToolTip, userData);
+                clone.OverlayText = item.OverlayText;
```

Comment:
`MyGuiGridItem` has no `OverlayText` member. Have you tried to use a tool to find the `MyGuiGridItem` class and read its source code?